src/logic/calculate.js

```diff
@@ -0,0 +1,129 @@
+import operate from './operate';
+
+function isNumber(item) {
+  return !!item.match(/[0-9]+/);
+}
+
+/**
+ * Given a button name and a calculator data object, return an updated
+ * calculator data object.
+ *
+ * Calculator data object contains:
+ *   total:s      the running total
+ *   next:String       the next number to be operated on with the total
+ *   operation:String  +, -, etc.
+ */
+export default function calculate(obj, buttonName) {
+  if (buttonName === 'AC') {
+    return {
+      total: null,
+      next: null,
+      operation: null,
+    };
+  }
```

Comment:
- [ ] I suggest that when you click on this button `AC` to display a default `zero in the screen

Reply:
thanks @DeliceLydia, I just needed to set the default of result as 0 in `calculate.js` 👍🏾